### DISCUSSION TITLE
Is aria-roledescription prohibited where the computedRole is generic?

### DISCUSSION THREAD

**Discussion by aleventhal** [w3c/aria#1469] [Q&A]
When a role="region" has no name, the computed role should be "generic".
At least I believe this to be true, and I'm working to fix this in Chrome now.

However, aria-roledescription is prohibited on role=generic

So is the aria-roledescription prohibited on nameless role=region or not?
I guess it depends on whether the rules applied are for the computed role or the marked up role.

**Comment by aleventhal** [comment]
For now, in Chrome I decided that aria-roledescription is not allowed for a nameless region, because the role has become "generic".
But we can always change it.

**Reply by scottaohara** [reply]
would agree with this.  Would expect similar for other elements/roles which are not exposed if they are not named.

**Reply by scottaohara** [reply]
seems like it would also make sense to not allow it / expose it for elements that have no implicit ARIA role.... in _most_ cases.
